pkg/odo/cli/service/list.go

```diff
@@ -48,7 +48,7 @@ func (o *ServiceListOptions) Complete(name string, cmd *cobra.Command, args []st
 			return err
 		}
 	} else {
-		return fmt.Errorf("failed to list operator backed services, have you installed operators on the cluseter?")
+		return fmt.Errorf("failed to list operator backed services, have you installed operators on the cluster?")
```

Comment:
Error string should not start with a capital letter or end in a punctuation mark, do you think we should change the message to the following?
```suggestion
		return fmt.Errorf("failed to list Operator backed services, make sure you have installed the Operators on the cluster")
```

Reply:
Makes senes; committed your suggestion.